doc/rtd/topics/bugs.rst

```diff
@@ -0,0 +1,95 @@
+.. _reporting_bugs:
+
+Reporting Bugs
+**************
+
+Logs
+====
+
+To aid in debugging collecting the necessary logs, first run the
+`collect-logs` subcommand to produce a tarfile that you can easily upload
+to aid in intial debugging of your issue:
+
+.. code-block:: shell-session
+
+  $ cloud-init collect-logs
+  Wrote /home/ubuntu/cloud-init.tar.gz
+
+If your cloud-init does not have the  `collect-logs` subcommand, then please
+manually collect the base log files by doing the following:
+
+.. code-block:: shell-session
+
+  $ dmesg > dmesg.txt
+  $ sudo journalctl -o short-precise > journal.txt
+  $ sudo tar -cvf cloud-init.tar dmesg.txt journal.txt /run/cloud-init \
+      /var/log/cloud-init.log /var/log/cloud-init-output.log
+
+Reporting
+=========
+
+Bugs for upstream cloud-init are tracked using Launchpad. To file a bug,
+`create a Launchpad account`_, login, and then
+`report an upstream cloud-init bug`_.
+
+If your bug is for a specific distro using cloud-init, please first consider
+reporting it with the upstream distro or confirm that it still occurs
+with the latest upstream cloud-init code. See below for details on specific
+distro reporting.
+
+Distro Specific Issues
+======================
+
+For issues specific to your distro please use one of the following distro
+specific reporting mechanisms:
+
+Ubuntu
+------
+
+To report a bug on Ubuntu use the `ubuntu-bug` command to automatically
```

Comment:
Should we make it clear that people should run it on an affected system (if possible)?

Reply:
Yes, done